core/pom.xml

```diff
@@ -46,6 +50,29 @@
             <version>${forestflow-latest.version}</version>
         </dependency>
 
+        <!-- https://doc.akka.io/docs/alpakka/current/s3.html#artifacts -->
+        <dependency>
+            <groupId>com.lightbend.akka</groupId>
+            <artifactId>akka-stream-alpakka-s3_${scala.binary.version}</artifactId>
+            <version>${akka-stream-alpakka-s3.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.typesafe.akka</groupId>
+            <artifactId>akka-stream_${scala.binary.version}</artifactId>
+            <version>${akka.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.typesafe.akka</groupId>
+            <artifactId>akka-http_${scala.binary.version}</artifactId>
+            <version>${akka.http.version}</version>
+        </dependency>
```

Comment:
I believe akka-http is already defined in the parent POM. Please double-check and be careful with introducing dependencies. Also the version used is already a property in the parent pom.